### PR TITLE
Quickstart Python draft

### PR DESCRIPTION
**Pull request by DennisDawson**
Early WIP for Python version of Quickstart tutorials.

### CODE REVIEW

**Comment by maria-robobug** [comment]
Hey @DennisDawson! Just having a quick glance (will try to look more into this tomorrow) - I believe Python uses snake case for method/function names.

Example:

```
get_standby_account()
```

The [PEP8 style guide](https://peps.python.org/pep-0008/) is also quite handy as a reference.

**Review by oeggert — APPROVED** [review]
Only suggestion is to move all the imports to the header/beginning. Though I can also see the case for keeping them with their related functions for ease of learning.

File: content/tutorials/quickstart/py-create-accounts-send-xrp.md [hunk 1]
```diff
@@ -215,21 +215,21 @@ import json
...
 ```python
-from mod1 import getAccount
-from mod1 import getAccountInfo
-from mod1 import sendXRP
+from mod1 import get_account
+from mod1 import get_accountInfo
+from mod1 import send_xrp
 ```
 
 ### getStandbyAccount
```
Comment (maria-robobug) [inline content/tutorials/quickstart/py-create-accounts-send-xrp.md:214]: Just a minor one, change the `getStandbyAccount` heading to `get_standby_account`.

File: content/tutorials/quickstart/py-create-accounts-send-xrp.md [hunk 2]
```diff
@@ -0,0 +1,437 @@
...
+### get_account
+
+This method lets you get an existing account by providing a seed value. If you provide no seed value, the method creates a new account for you.
+
+Import required methods.
+
+```python
+def get_account(_seed):
+    from xrpl.clients import JsonRpcClient
+    from xrpl.wallet import Wallet
```
Comment (maria-robobug) [inline content/tutorials/quickstart/py-create-accounts-send-xrp.md]: I think it's better practice to place any imports at the top of the file, plus we avoid repetition.

File: content/tutorials/quickstart/py-create-accounts-send-xrp.md [hunk 3]
```diff
@@ -0,0 +1,437 @@
...
+## mod1.py
+
+The mod1.py module contains the business logic for interacting with the XRP Ledger.
+
+Import the XRPL and JSON libraries.
+
+```python
+import xrpl
+import json
+```
```
Comment (maria-robobug) [inline content/tutorials/quickstart/py-create-accounts-send-xrp.md]:
```python
import xrpl
import json

from xrpl.clients import JsonRpcClient
from xrpl.wallet import Wallet
from xrpl.models.requests.account_info import AccountInfo
```

**Review by maria-robobug — APPROVED** [review]
👍🏾 LGTM - I would also suggest moving the module imports to the top of the file. See https://peps.python.org/pep-0008/#imports

**Comment by ckniffen** [comment]
Just warning you that the DGE team is planning on releasing 2.0 for Python soon.  There will be a few breaking changes including the AccountSetFlag enum. https://github.com/XRPLF/xrpl-py/pull/524.

**Pull request merged**